Sources/BraintreeThreeDSecure/BTThreeDSecureV2Provider.m

```diff
@@ -1,7 +1,7 @@
 #import "BTThreeDSecureV2Provider.h"
 #import "BTPaymentFlowDriver+ThreeDSecure_Internal.h"
 #import "BTThreeDSecureAuthenticateJWT.h"
-#import <CardinalMobile/CardinalMobile.h>
+#import "BTProxyCardinal.h"
```

Comment:
thought: is the prefix considered to be `BTProxy` for this addition? It does help indentify the pattern, though I wonder if `BTCardinal` would work standalone to simplify it for the merchant.

Reply:
It shouldn't matter to merchants -- the proxy protocols are only used internally. I kinda like including the word "proxy" to help explain what's going on.